e2e/test/scenarios/embedding/embedding-smoketests.cy.spec.js

```diff
@@ -330,8 +331,16 @@ function assertLinkMatchesUrl(text, url) {
 
 function ensureEmbeddingIsDisabled() {
   openSharingMenu();
-  // XXX: Check the embed modal instead when implemented
-  // sharingMenu().findByText(/embedding is off/i);
+  sharingMenu()
```

Comment:
Implement the assertion disabled in https://github.com/metabase/metabase/pull/50513